Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialIntegratedPrimary.mo

```diff
@@ -89,40 +89,50 @@ model PartialIntegratedPrimary
     final y_start=yVal6_start)
     "Bypass valve: closed when free cooling mode is deactivated;
     open when free cooling is activated"
-    annotation (Placement(transformation(extent={{-40,-30},{-60,-10}})));
+    annotation (Placement(transformation(extent={{-30,-30},{-50,-10}})));
 
+  Fluid.FixedResistances.Junction spl2(
+    redeclare package Medium = Medium2,
+    m_flow_nominal={numChi*m2_flow_chi_nominal,-numChi*m2_flow_chi_nominal,-
+        m2_flow_wse_nominal},
```

Comment:
`-m2_flow_wse_nominal`

Reply:
Negative sign is there (it is just on the line above). Formatting fixed for better clarity.